app/screens/Library/index.js

```diff
@@ -11,6 +12,7 @@ import Book from './components/Book';
 const LibraryWithLoading = withLoading(Library);
 class LibraryContainer extends Component {
   componentDidMount() {
+    StatusBar.setBackgroundColor('rgba(0,0,0,0.3)');
```

Comment:
it should be transparent

Reply:
The card requires 30% opacity :crying_cat_face: 

![image](https://user-images.githubusercontent.com/38475587/70075953-90875c80-15dc-11ea-83e0-d3e8b4c5f620.png)